modules/fabric-net-firewall/main.tf

```diff
@@ -131,7 +131,7 @@ resource "google_compute_firewall" "custom" {
   target_service_accounts = each.value.use_service_accounts ? each.value.targets : null
   disabled                = lookup(each.value.extra_attributes, "disabled", false)
   priority                = lookup(each.value.extra_attributes, "priority", 1000)
-  enable_logging          = lookup(each.value.extra_attributes, "enable_logging", null)
+  log_config              = lookup(each.value.extra_attributes, "log_config", null)
```

Comment:
We should use a dynamic block like here
https://github.com/terraform-google-modules/terraform-google-network/blob/1a2d1b5c3badb58a87ae845afa2123f26f64d093/modules/subnets/main.tf#L34